great-idea/css/index.css

```diff
@@ -63,4 +63,167 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+h1, .services {
+	font-size: 30px;
+	margin-top: 5%;
+}
+
+nav {
+	padding: 30px 0;
+	display: flex;
+}
+nav a {
+	margin-right: 60px;
+	text-decoration: none;
+	color: #999;
+	display: flex;
+	width: 80px;
+	justify-content: center;
+}
+h1.index {
+	font-size: 80px;
+	text-align: center;
+}
+h2 {
+	font-size: 20px;
+}
+p {
+	font-size: 18px;
+}
+.container {
+	max-width: 1024px;
+	margin: 0 auto;
+	height: 1536px;
+}
+.contents {
+	height: 400px;
+	display: flex;
+	margin-top: 60px;
+}
+.top-left {
+	display: flex;
+	width: 38%;
+	height: 400px;
+	flex-direction: column;
+	justify-content: center;
+	align-items: center;
+}
+.top-right {
+	display: flex;
+	/*width: 49%;*/
+	height: 400px;
+	margin-left: auto;
+}
+.top-content {
+	width: 100%;
+	display: flex;
+}
+.card {
+	margin-top: 20px;
+}
+.card2 {
+	margin-top: 20px;
+	margin-left: 70px;
+}
+.card4, .card5 {
+	margin-top: 20px;
+	margin-left: 45px;
+}
+.bottom-content{
+	display: flex;
+}
+.logo {
+	margin-left: 100px;
+	display: flex;
+}
+.middle-img {
+	margin-top: 5%;
+	margin-bottom: 2%;
+	width: 100%;
+}
+.contact {
+	margin-top: 5%;
+}
+hr {
+	margin-top: 5%;
+	color: black;
+}
+hr.line {
+	margin-top: 4%;
+	color: black;
+}
+footer p {
+	margin: 0 auto;
+	width: 20%
+}
+.btn {
+	background-color: white;
+    border: 1px solid black;
+    color: black;
+    padding: 15px 32px;
+    text-decoration: none;
+	display: flex;
+    font-size: 16px;
+	cursor: pointer;
+	height: 32px;
+	width: 45%;
+	justify-content: center;
+	align-items: center;
+}
+.one {
```

Comment:
try to use a better naming convention with your classes. you would want somebody else to look at your code and understand what each part is